src/api/resource-service-api/concepts.ts

```diff
@@ -0,0 +1,30 @@
+import { resourceServiceApiGet } from './host';
+import { Concept } from '../../types';
+
+const mapSorting = ({ sortfield }: any) =>
+  sortfield === 'harvest.firstHarvested'
+    ? { field: 'harvest.firstHarvested', direction: 'desc' }
+    : undefined;
+
+export const searchConcepts = (body: any) =>
+  resourceServiceApiGet('/concepts', body?.filters[0]);
```

Comment:
Det er ikke et søk og GET-metoden håndterer ikke body
```suggestion
export const getConcepts = () =>
  resourceServiceApiGet('/concepts');
```